fpdf/drawing.py

```diff
@@ -0,0 +1,1023 @@
+from contextlib import contextmanager
+import enum
+import math
+
+
+def _clamp8(value):
+    return 0.0 if value < 0.0 else 255.0 if value > 255.0 else value
+
+
+def _render_number(number):
+    # this approach tries to produce minimal representations of floating point numbers
+    # but can also produce "-0".
+    return ("%.4f" % number).rstrip("0").rstrip(".")
+
+
+class Color:
+    def __init__(self, parameters):
+        self._parameters = parameters
+
+    def render(self):
+        return " ".join(_render_number(param) for param in self._parameters)
+
+
+class RGBColor(Color):
+    operand = "rg"
```

Comment:
`pylint` will probably complain here about this class constant not being UPPERCASE

Reply:
I'm not seeing that as one of the things it's complaining about when I run it locally, so I should probably check my `pylint` invocation to make sure I've got the settings right. I believe it does make sense to uppercase this, though.

Reply:
OK, then I may be even more rigid than `pylint` here ^^

Reply:
This particular case has been fixed and I've tried to follow this convention elsewhere, but another once-over is probably in order to make sure I didn't miss anything.